test/Microsoft.ComponentDetection.Detectors.Tests/PubComponentDetectorTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace Microsoft.ComponentDetection.Detectors.Tests;
+
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.ComponentDetection.Contracts;
+using Microsoft.ComponentDetection.Contracts.TypedComponent;
+using Microsoft.ComponentDetection.Detectors.Pub;
+using Microsoft.ComponentDetection.TestsUtilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+[TestClass]
+[TestCategory("Governance/All")]
+[TestCategory("Governance/ComponentDetection")]
+public class PubComponentDetectorTests : BaseDetectorTest<PubComponentDetector>
```

Comment:
Could we also add verification tests? should be in the `tests/Microsoft.ComponentDetection.VerificationTests/resources`